src/settings.ts

```diff
@@ -566,3 +566,5 @@ export const POSTERPAGE_MAX_VIDEO_PARTICIPANTS = 10;
 export const SEARCH_DEBOUNCE_TIME = 200; // ms
 
 export const DEFAULT_DISPLAYED_POSTER_PREVIEW_COUNT = 12;
+
+export const START_DATE = "conference"; // options: "conference", "course", "today"
```

Comment:
A little hacky (I would usually extract this into a type on it's own), but you can use TypeScript to restrict this to the values you want.

```suggestion
export const START_DATE: "conference" | "course" | "today" = "conference";
```